util/src/main/java/org/kaazing/gateway/util/InternalSystemProperty.java

```diff
@@ -133,6 +133,8 @@
 
     UPDATE_CHECK_SERVICE_URL("org.kaazing.gateway.server.UPDATE_CHECK_SERVICE_URL"),
 
+    CLUSTER_BYPASS_AWS_CHECK("com.kaazing.cluster.bypass.aws.check", "false"),
```

Comment:
Thanks, It should be "com.kaazing.gateway" prefix so please make it "com.kaazing.gateway.cluster.bypass.aws.check"